ci/jjb/jobs/disk-image-builder.yaml

```diff
@@ -5,6 +5,8 @@
         - qe-ownership
     scm:
         - pulp-ci-github
+    triggers:
+        - timed: 'H H * * 6'
```

Comment:
Is this being replaced by https://github.com/pulp/pulp-ci/pull/567 ?

Reply:
Hi @rochacbruno ,

#567 was merged without scheduling disk image builder job. This commit will do that. The jobs will be scheduled to run every Saturday replacing the existing images on OpenStack.
 
Do let me know if this answers your question.